File: 1475b/main.py
def I():
    return input()


def II():
    return int(input())


def MII():
    return map(int, input().split())


def LI():
    return list(input().split())


def LII():
    return list(map(int, input().split()))

a = II()
for _ in range(a):
    b = II()
    c = b // 2020

    if c == 0:
        print("NO")
    elif c*2020 <= b <= c*2021:
        print("YES")
    else:
        print("NO")

File: 1475b/main.go
package main

import "fmt"

func main() {
	var a int
	fmt.Scan(&a)
	for i := 1; i <= a; i++ {
		var b, c int
		fmt.Scan(&b)
		c = b / 2020
		//fmt.Println(b, c)
		if c == 0 {
			fmt.Println("NO")
		} else if c*2020 <= b && b <= c*2021 {
			fmt.Println("YES")
		} else {
			fmt.Println("NO")
		}
	}
}
